tmt/steps/provision/mrack.py

```diff
@@ -1458,3 +1463,22 @@ def guest(self) -> Optional[GuestBeaker]:
         """
 
         return self._guest
+
+
+@container
+class GuestLogBeaker(tmt.steps.provision.GuestLog):
+    def __init__(self, guest: GuestBeaker, name: str, url: Optional[str] = None) -> None:
```

Comment:
With `container`, you shouldn't need custom `__init__()`, one will be generated automagically for you.